src/js/utils/ValidatorUtil.js

```diff
@@ -34,22 +34,22 @@ var ValidatorUtil = {
   },
 
   isInteger: function (value) {
-    const number = parseFloat(value);
-
-    return Number.isInteger(number);
+    return ValidatorUtil.isNumber(value) &&
+      Number.isInteger(parseFloat(value));
```

Comment:
Are you ok doing this in a different PR as I'd like to update the other validator utils names as well?